src/components/BottomSheet/BottomSheet.jsx

```diff
@@ -9,6 +9,7 @@ const makeStyles = ({ isTopPosition }) => ({
     borderTopLeftRadius: '1rem',
     borderTopRightRadius: '1rem',
     transition: 'border-radius 0.5s',
+    zIndex: 'var(--zIndex-drawer)',
```

Comment:
You like the danger. Are you considering the BS as a drawer?

Reply:
Oui, le drawer c'est pas le menu qui pop depuis le bas de l'écran ?